<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilesPage</class>
 <widget class="QWidget" name="FilesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>611</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="startupBox">
     <property name="title">
      <string>Startup Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="presetControls">
        <item>
         <widget class="QPushButton" name="addPreset">
          <property name="toolTip">
           <string>Saving the current project as a preset</string>
          </property>
          <property name="whatsThis">
           <string>Saving the current project as a preset</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removePreset">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setDefaultPreset">
          <property name="text">
           <string>Make Default</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="presetListWidget">
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="askPresetRbtn">
        <property name="text">
         <string>Ask on startup</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">startupGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="loadDefaultPresetRbtn">
        <property name="text">
         <string>Load default preset</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">startupGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="loadLastActiveRbtn">
        <property name="text">
         <string>Load last active file</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">startupGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="autosaveBox">
     <property name="title">
      <string comment="Preference">Autosave documents</string>
     </property>
     <layout class="QVBoxLayout" name="lay">
      <item>
       <widget class="QCheckBox" name="autosaveCheckBox">
        <property name="text">
         <string comment="Preference">Enable autosave</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="autosaveNumberLabel">
        <property name="text">
         <string comment="Preference">Number of modifications before autosaving:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="autosaveNumberBox">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>8192</number>
        </property>
        <property name="value">
         <number>256</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="startupGroup"/>
 </buttongroups>
</ui>
